Trying:
    say_my_name = __import__('3-say_my_name').say_my_name
Expecting nothing
ok
Trying:
    say_my_name(35, "Black")
Expecting:
    Traceback (most recent call last):
    TypeError: first_name must be a string
ok
Trying:
    say_my_name("John", 35)
Expecting:
    Traceback (most recent call last):
    TypeError: last_name must be a string
ok
Trying:
    say_my_name("John", "Black")
Expecting:
    My name is John Black
ok
Trying:
    say_my_name()
Expecting nothing
**********************************************************************
File "./tests/3-say_my_name.txt", line 20, in 3-say_my_name.txt
Failed example:
    say_my_name()
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python3.12/doctest.py", line 1361, in __run
        exec(compile(example.source, filename, "single",
      File "<doctest 3-say_my_name.txt[4]>", line 1, in <module>
        say_my_name()
    TypeError: say_my_name() missing 1 required positional argument: 'first_name'
**********************************************************************
1 items had failures:
   1 of   5 in 3-say_my_name.txt
5 tests in 1 items.
4 passed and 1 failed.
***Test Failed*** 1 failures.
